### ISSUE TITLE
Cannot select problems option in wizard

### ISSUE THREAD

**Issue by steinybot** [eslint/eslint#13937]
**Tell us about your environment**

* **ESLint Version:** v7.15.0
* **Node Version:** v14.15.2
* **npm Version:** v6.14.9

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?**

N/A

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js

```

```bash
yarn run eslint --init
```

**What did you expect to happen?**

When I use the arrow keys to select the option "To check syntax, find problems, and enforce code style" and press enter it should select that option.

**What actually happened? Please include the actual, raw output from ESLint.**

It selects the "style" option instead.

If I press space rather than the down arrow then it doesn't change the highlighted line but it does select the correct option.

![eslint](https://user-images.githubusercontent.com/4659562/102539645-d1ccaa80-4112-11eb-8412-e692223da7de.gif)

**Are you willing to submit a pull request to fix this bug?**

**Comment by mdjermanovic** [comment]
Hi @steinybot, thanks for the issue!

> **What did you expect to happen?**
> 
> When I use the arrow keys to select the option "To check syntax, find problems, and enforce code style" and press enter it should select that option.
> 
> **What actually happened? Please include the actual, raw output from ESLint.**
> 
> It selects the "style" option instead.

This is the expected behavior, "style" is short for "To check syntax, find problems, and enforce code style".

"To check syntax" means to run the parser, which is always included.

"find problems" corresponds to `"extends": "eslint:recommended"`, which will be added to the config file, unless you choose "Use a popular style guide" (since the style guides already provide their configurations to find problems).

In some of the following steps, you should see a question on how to define the style:

```
? How would you like to define a style for your project? ...
> Use a popular style guide
  Answer questions about your style
  Inspect your JavaScript file(s)
```
